src/app/components/core/user-management/user-management.component.ts

```diff
@@ -168,7 +168,7 @@ export class UserManagementComponent implements OnInit, OnDestroy {
     } else if (userData.key == 'enableUser') {
       this.enableUser(userData.id);
     } else if (userData.key == 'clearSessions') {
-      this.clearAllSessions(userData.key);
+      this.clearAllSessions(userData);
```

Comment:
only `userData.id` can be passed to `clearAllSessions` function

Reply:
That is passed to manage the data on the component only. In case we have two method to manage on same component we need key as well. Only for consistency we are passing the full data on every component.